src/test/java/org/pesmypetcare/webservice/dao/communitymanager/ForumDaoTest.java

```diff
@@ -292,19 +292,19 @@ public void createForum() throws DatabaseAccessException, DocumentException {
             public void updateTags() throws DatabaseAccessException, DocumentException {
                 mockGetGroupAndForumIds();
                 willDoNothing().given(documentAdapter)
-                    .updateDocumentFields(anyString(), anyString(), any(FieldValue.class), any(WriteBatch.class));
+                    .updateDocumentFields(any(WriteBatch.class), anyString(), anyString(), any(FieldValue.class));
                 mockAddForumToTag();
 
                 dao.updateTags(groupName, forumName, tags, tags);
                 verify(documentAdapter)
-                    .updateDocumentFields(eq(forumPath), eq("tags"), eq(FieldValue.arrayRemove(tags.toArray())),
-                        same(batch));
+                    .updateDocumentFields(same(batch), eq(forumPath), eq("tags"),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "tags" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5649852)